src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/Publisher.cs

```diff
@@ -3,9 +3,8 @@
 using Microsoft.Azure.WebJobs;
 using Microsoft.Extensions.Logging;
 
-namespace GovUk.Education.ExploreEducationStatistics.Publisher
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Functions
 {
-    // ReSharper disable once UnusedMember.Global
```

Comment:
The two ReSharper disable statements removed here were useful so I'm not sure why they are removed